build/cqs.tsv

```diff
@@ -11,7 +11,7 @@ com.esotericsoftware:kryo-shaded	3.0.3	compile
 com.esotericsoftware:minlog	1.3.0	compile
 com.fasterxml.jackson.core:jackson-annotations	2.9.10	compile
 com.fasterxml.jackson.core:jackson-core	2.9.10	compile
-com.fasterxml.jackson.core:jackson-databind	2.9.10.3	compile
+com.fasterxml.jackson.core:jackson-databind	2.9.10.4	compile
```

Comment:
should be covered as a bug-fix patch version. I didn't actually change this in this PR, I think we forgot to run the cqs when we upgraded it